drivers/gles3/storage/material_storage.cpp

```diff
@@ -2421,8 +2421,8 @@ void MaterialStorage::shader_set_code(RID p_shader, const String &p_code) {
 			shader->data = nullptr;
 		}
 
-		for (RBSet<Material *>::Element *E = shader->owners.front(); E; E = E->next()) {
-			Material *material = E->get();
+		for (Material *E : shader->owners) {
+			Material *material = E;
```

Comment:
Ideally stuff like this should be this:
```suggestion
		for (Material *material  : shader->owners) {
```
Not sure if I should bother to try and fix all the instances where something like this happens in this PR though. I think it'd be better suited for a future PR, checking for all types range iterators.

Reply:
I guess that could be a follow-up PR yeah, as long as it's not too hard to find those spots after the pact. Maybe grepping for ` = E;` can help find such locations.